xclim/indices/_threshold.py

```diff
@@ -717,6 +719,68 @@ def maximum_consecutive_tx_days(
     return group.apply(rl.longest_run, dim="time")
 
 
+@declare_units("[area]", sic="[]", area="[area]", thresh="[]")
+def sea_ice_area(sic, area, thresh="15 pct"):
+    """Return the total sea ice area.
+
+    Sea ice area measures the total sea ice covered area where sea ice concentration is above a threshold,
```

Comment:
I think I need a much clearer description between the sea_ice_area vs sea_ice_extent.   Probably super clear for people who know the subject better than myself but still could be helpful

Reply:
http://nsidc.org/arcticseaicenews/faq/#area_extent

Reply:
Is this link official / stable enough to add in the notes for these two indicators?

Reply:
If you want to play it safe, you could use the site image from archive.org (the wayback machine):

https://web.archive.org/web/20200103090158/https://nsidc.org/arcticseaicenews/faq/#area_extent

This is an archived image of the page according to the date and time specified (2020-01-03 09:01:58). That's about as stable as it gets.